### ISSUE TITLE
Set a description for built-in clients and make sure it is shown in the client list

### ISSUE THREAD

**Issue by pedroigor** [keycloak/keycloak#25303]
### Description

When looking at the client list is not clear what the purpose for the built-in clients because there is no description explaining their purpose and what they stand for.

For instance, for `account-console` explain it is the client used to access the account console. The same goes for other clients.

### Motivation

Improve UX.

**Comment by ssilvert** [comment]
I agree, but this is actually not a UI issue.  These clients are created on the back end when a new realm is created.
